### PR TITLE
Added ezplatform-personalization to composer.json

### PR DESCRIPTION
**Pull request by ciastektk**

### CODE REVIEW

**Comment by andrerom** [comment]
Why referring to git as repo here?

**Comment by lserwatka** [comment]
It is a mistake. We need to add this repo to satis.

File: composer.json [hunk 1]
```diff
@@ -14,6 +14,10 @@
         {
             "type": "composer",
             "url": "https://updates.ez.no/ttl"
+        },
+        {
+            "type": "git",
+            "url": "https://github.com/ezsystems/ezplatform-personalization.git"
```
Comment (SylvainGuittard) [inline composer.json:20]: This configuration does not work on Ibexa Cloud
The following code works perfectly.
```suggestion
            "type": "vcs",
            "url": "https://github.com/ezsystems/ezplatform-personalization"
```

**Review by SylvainGuittard — CHANGES_REQUESTED** [review]

File: composer.json (hunk 1 above)
Comment (andrerom) [reply composer.json:20]: Should rather add it here: https://github.com/ezsystems/support-site/blob/master/updates-app/satis.json

Then we can just swap this repo to start to use updates.ibexa.co for master in order to start to actively test it.
See #176

Comment (andrerom) [reply composer.json:20]: Solving this in link below for updates.ibexa.co

Comment (SylvainGuittard) [reply composer.json:20]: I commented without looking at the discussion first. This PR was shared with me 2 days ago so I can install it on p.sh. 
I agree it should be added to Satis. All good for me.

**Pull request closed without merging**